File: EuroTrains/ClientApp/src/app/api/models/booking-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from '../models/time-place-rm';
export interface BookingRm {
  arrival?: TimePlaceRm;
  company?: string | null;
  departure?: TimePlaceRm;
  numberOfBookedSeats?: number;
  passengerEmail?: string | null;
  price?: string | null;
  trainId?: string;
}

File: EuroTrains/ReadModels/BookingRm.cs
namespace EuroTrains.ReadModels
{
    public record BookingRm(
         Guid TrainId,
         string Company,
         string Price,
         TimePlaceRm Arrival,
         TimePlaceRm Departure,
         int NumberOfBookedSeats,
         string PassengerEmail);
}
